catalog-entities/marketplace/packages/backstage-plugin-catalog-backend-module-bitbucket-cloud.yaml

```diff
@@ -26,7 +27,7 @@ spec:
   support: tech-preview
   lifecycle: active
   partOf:
-    - backstage-bitbucket-cloud-catalog-module
+    - bitbucket-cloud-catalog-integration
```

Comment:
I agree, I can not find plugin with name: `backstage-bitbucket-cloud-catalog-module`. You proposed plugin `bitbucket-cloud-catalog-integration`. But when I take a look plugin content, I can not find package with name`backstage-plugin-catalog-backend-module-bitbucket-cloud`  in the packages section....